Evaluations.js

```diff
@@ -696,7 +703,7 @@ function sendEvaluationReminderEmails() {
     const reviewLogSheet = SpreadsheetApp.openById(AMBASSADOR_REGISTRY_SPREADSHEET_ID).getSheetByName(
       REVIEW_LOG_SHEET_NAME
     );
-    const formResponseSheet = SpreadsheetApp.openById(AMBASSADORS_SCORES_SPREADSHEET_ID).getSheetByName(
+    const formResponseSheet = SpreadsheetApp.openById(AMBASSADORS_SUBMISSIONS_SPREADSHEET_ID).getSheetByName(
```

Comment:
Used to be Ambassadors_Scores - need to re-verify Submissions the correct sheet?

Reply:
no no these all are ok

Reply:
no, all these sheets references are correct